tools/omero/omero_metadata_upload/omero_metadata_import.xml

```diff
@@ -0,0 +1,126 @@
+<tool id="omero_metadata_import" name="OMERO Metadata Import" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@"
+      profile="20.01" license="MIT">
+    <description> with ezomero </description>
+    <macros>
+        <token name="@TOOL_VERSION@">5.18.0</token>
+        <token name="@VERSION_SUFFIX@">0</token>
+    </macros>
+    <xrefs>
+        <xref type="bio.tools">omero</xref>
+    </xrefs>
+    <requirements>
+        <requirement type="package" version="3.0.1">ezomero</requirement>
+        <requirement type="package" version="2.2.2">pandas</requirement>
+        <requirement type="package" version="21.0.2">openjdk</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        python $__tool_directory__/omero_metadata_upload.py
+        --user $__user__.extra_preferences.get('omero|username', $test_username)
+        --pws $__user__.extra_preferences.get('omero|password', $test_password)
+        --host $omero_host
+        --port $omero_port
+        --obj_type $obj_type
+        --ann_type $ann_type
+        --ann_file $ann_file
+        --an_name $an_name
+        --log_file $log
+        #if $object_id_selection == "existing_object"
+            --did "$did"
+        #end if
+    ]]></command>
+    <inputs>
+        <param argument="omero_host" type="text" optional="false" label="host" help="OMERO host"/>
+        <param argument="omero_port" type="integer" optional="false" value="4064" label="port" help="OMERO port"/>
+        <param argument="obj_type" type="select" optional="true" label="OMERO Object" help="Select the target OMERO object">
+            <option value="project">Project</option>
+            <option value="screen">Screen</option>
+            <option value="dataset">Dataset</option>
+            <option value="image">Image</option>
+        </param>
+        <conditional name="object_id">
+        <param name="object_id_selection" type="select" help="Create a new OMERO object or target an existing one">
+            <option value="new_object">Create new OMERO object</option>
+            <option value="existing_object">Target an existing OMERO object</option>
+        </param>
+        <when value="new_object">
+        </when>
+        <when value="existing_object">
+            <param name="did" type="integer" value="1409"/>
+        </when>
+        </conditional>
+        <param argument="ann_type" type="select" optional="false" label="Annotation type" help="Select annotation format">
+            <option value="table">An OMERO Table</option>
+            <option value="KV">OMERO Key-Value Pairs</option>
+        </param>
+        <param argument="ann_file" type="data" format="tabular" optional="false" label="Annotation file" help="Select annotation file"/>
```

Comment:
Can you document how such a table looks.

Reply:
Done, I had a help section in the .xml